SocialCareCaseViewerApi/V1/Gateways/DatabaseGateway.cs

```diff
@@ -905,6 +905,34 @@ public Person GetPersonWithPersonalRelationshipsByPersonId(long personId, bool i
             return personWithRelationships;
         }
 
+        public PersonalRelationshipType GetPersonalRelationshipTypeByDescription(string description)
+        {
+            return _databaseContext.PersonalRelationshipTypes
+                .FirstOrDefault(prt => prt.Description == description);
```

Comment:
should we care about case? You might want to call `.ToLower()` against both strings.

Reply:
I guess we shouldn't really. Will add dis!

Reply:
Done in https://github.com/LBHackney-IT/social-care-case-viewer-api/pull/345/commits/e07b2c23bca6bb074d9c0ffb99407cd19731d6e1!